packages/web-components/src/menu-list/menu-list.spec.ts

```diff
@@ -0,0 +1,547 @@
+                    <fluent-menu-item role="menuitemradio">Menu item 2</fluent-menu-item>
+                    <fluent-divider role="separator"></fluent-divider>
+                    <fluent-menu-item role="menuitemradio">Menu item 3</fluent-menu-item>
+                    <fluent-menu-item role="menuitemradio">Menu item 4</fluent-menu-item>
+                </fluent-menu-list>
+            `;
+    });
+
+    await menuItems.nth(0).click();
+
+    await expect(menuItems.nth(0)).toHaveAttribute('aria-checked', 'true');
+    await expect(menuItems.nth(1)).toHaveAttribute('aria-checked', 'false');
+    await expect(menuItems.nth(2)).toHaveAttribute('aria-checked', 'false');
+    await expect(menuItems.nth(3)).toHaveAttribute('aria-checked', 'false');
+
+    await menuItems.nth(1).click();
+
+    await expect(menuItems.nth(0)).toHaveAttribute('aria-checked', 'false');
+    await expect(menuItems.nth(1)).toHaveAttribute('aria-checked', 'true');
+    await expect(menuItems.nth(2)).toHaveAttribute('aria-checked', 'false');
+    await expect(menuItems.nth(3)).toHaveAttribute('aria-checked', 'false');
+
+    await menuItems.nth(2).click();
+
+    await expect(menuItems.nth(0)).toHaveAttribute('aria-checked', 'false');
+    await expect(menuItems.nth(1)).toHaveAttribute('aria-checked', 'true');
+    await expect(menuItems.nth(2)).toHaveAttribute('aria-checked', 'true');
+    await expect(menuItems.nth(3)).toHaveAttribute('aria-checked', 'false');
+
+    await menuItems.nth(3).click();
+
+    await expect(menuItems.nth(0)).toHaveAttribute('aria-checked', 'false');
+    await expect(menuItems.nth(1)).toHaveAttribute('aria-checked', 'true');
+    await expect(menuItems.nth(2)).toHaveAttribute('aria-checked', 'false');
+    await expect(menuItems.nth(3)).toHaveAttribute('aria-checked', 'true');
+  });
+
+  test('should navigate the menu on arrow up/down keys', async () => {
+    await root.evaluate(node => {
+      node.innerHTML = /* html */ `
+                <fluent-menu-list>
+                    <fluent-menu-item>Menu item 1</fluent-menu-item>
+                    <fluent-menu-item>Menu item 2</fluent-menu-item>
+                    <fluent-menu-item>Menu item 3</fluent-menu-item>
+                    <fluent-menu-item>Menu item 4</fluent-menu-item>
+                </fluent-menu-list>
+            `;
+    });
+
+    await element.waitFor({ state: 'attached' });
+
+    await element.evaluate(node => {
+      node.focus();
+    });
+
+    await expect(menuItems).toHaveCount(4);
+
+    await expect(menuItems.first()).toBeFocused();
+
+    await element.press('ArrowDown');
+
+    await expect(menuItems.nth(1)).toBeFocused();
+
+    await element.press('ArrowDown');
+
+    await expect(menuItems.nth(2)).toBeFocused();
+
+    await element.press('ArrowDown');
+
+    await expect(menuItems.nth(3)).toBeFocused();
+  });
+
+  test('should close the menu when pressing the escape key', async () => {
```

Comment:
Looks like this is testing to see if the focus returns to the parent item after the Esc key is pressed. I would reword the test to make your assertion clearer.

Reply:
good catch